docs/components/Code.tsx

```diff
@@ -108,6 +110,14 @@ function LiveCode({
 		[toggleIsCodeVisible]
 	);
 
+	// LiveEditor doesn't support aria-label, so we have to do it the DOM way
+	useEffect(() => {
+		const pre = liveEditorRef.current?.querySelector('pre');
+		if (pre) {
+			pre.ariaLabel = `Live code editor ${id}`;
```

Comment:
Don't want to include the escape key helper?

Reply:
I don't know how valid it is, mobile SRs don't have to…